action.yml

```diff
@@ -20,9 +20,13 @@ inputs:
     description: "Okta MFA Method (https://developer.okta.com/docs/reference/api/factors/)"
     required: false
     default: "token:software:totp"
+  totp_time_slot:
+    description: "Time slot (in seconds) for which OTP is valid"
+    required: false
+    default: 5
```

Comment:
This now should be 10.
Btw, it would be great to fix the typo in `okta_passowrd` so we stop getting that misleading error.

Reply:
Fixed